### ISSUE TITLE
BUG - east update toolchain fails on project running under WSL 

### ISSUE THREAD

**Issue by michael-portmann** [IRNAS/irnas-east-software#65]
Completed all steps in document 'building_zephyr_fw_on_windows.md' up to step east update toolchain successfully.
Note 1. using latest versions of packages.
Note 2. Minor errors in document regarding setup of nRF Command Line Tools missing home directory ~ for /opt

mportmann@MPLAPTOP:~/sentek-iotprobe-firmware$ east update toolchain
Traceback (most recent call last):
  File "/home/mportmann/.local/bin/east", line 8, in <module>
    sys.exit(main())
  File "/home/mportmann/.local/lib/python3.10/site-packages/east/__main__.py", line 83, in main
    cli()
  File "/home/mportmann/.local/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/mportmann/.local/lib/python3.10/site-packages/rich_click/rich_group.py", line 21, in main
    rv = super().main(*args, standalone_mode=False, **kwargs)
  File "/home/mportmann/.local/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/mportmann/.local/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/mportmann/.local/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/mportmann/.local/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/mportmann/.local/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/mportmann/.local/lib/python3.10/site-packages/click/decorators.py", line 38, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/home/mportmann/.local/lib/python3.10/site-packages/east/workspace_commands/update_commands.py", line 44, in toolchain
    if east.ncs_version_installed:
AttributeError: 'EastContext' object has no attribute 'ncs_version_installed'

**Comment by michael-portmann** [comment]
A followup to the problem, I was able to manually install the required ncs version for the project
e.g. nrfutil-toolchain-manager.exe install v2.2.0
and then use east to build the project successfully.

**Comment by MarkoSagadin** [comment]
Hello @michael-portmann this issue is now resolved in v0.11.0.

I also updated the internal nrf toolchain manager binary. 
Bellow is the suggested code for upgrading:
```
pip install --upgrade east-tool
rm -fr ~/.local/share/east/nrfutil-toolchain-manager.exe
east sys-setup
```
